api/src/main/java/uk/gov/hmcts/fees/register/api/configuration/SpringSecurityConfiguration.java

```diff
@@ -18,53 +18,95 @@
 
 @Configuration
 @EnableWebSecurity
-public class SpringSecurityConfiguration extends WebSecurityConfigurerAdapter {
+public class SpringSecurityConfiguration {
 
-    private final AuthCheckerUserOnlyFilter authCheckerFilter;
+    private final RequestAuthorizer<User> userRequestAuthorizer;
+    private final AuthenticationManager authenticationManager;
 
-    @Autowired
-    public SpringSecurityConfiguration(final RequestAuthorizer<User> userRequestAuthorizer,
-                                       final AuthenticationManager authenticationManager) {
-        authCheckerFilter = new AuthCheckerUserOnlyFilter(userRequestAuthorizer);
-        authCheckerFilter.setAuthenticationManager(authenticationManager);
+    public SpringSecurityConfiguration(
+        RequestAuthorizer<User> userRequestAuthorizer,
+        AuthenticationManager authenticationManager
+    ) {
+        this.userRequestAuthorizer = userRequestAuthorizer;
+        this.authenticationManager = authenticationManager;
     }
 
-    @Override
-    public void configure(final WebSecurity web) {
```

Comment:
I think these endpoints can be added to the publicFilterChain method, since it includes `.anyRequest().permitAll()` it will replace web.ignoring().

Reply:
They are actually added there

Reply:
We discussed this is covered in line 47 and it was originally allowing public access to the endpoints publicly anyway so this is fine.